package.json

```diff
@@ -27,14 +24,18 @@
     "lint:js:fix": "npx eslint . --fix",
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "release": "multi-semantic-release --ignore-packages=packages/**",
-    "storage": "npm run dev --workspace=@pins/document-storage",
+    "storage": "npm run dev --workspace=@pins/applications.document-storage",
     "test": "turbo run test",
-    "tscheck": "turbo run prisma-generate --concurrency=1 && turbo run tscheck --parallel --filter=!@pins/api --filter=!@pins/appeals.api",
-    "web": "npm run dev --workspace=@pins/web",
+    "tscheck": "turbo run prisma-generate --concurrency=1 && turbo run tscheck --parallel --filter=!@pins/applications.api --filter=!@pins/appeals.api",
     "dev:appeals": "turbo run dev --filter=./appeals/* --no-cache --parallel --continue",
     "test:appeals": "turbo run test --filter=./appeals/*",
     "web:appeals": "npm run dev --workspace=@pins/appeals.web",
-    "api:appeals": "npm run dev --workspace=@pins/appeals.api"
+    "api:appeals": "npm run dev --workspace=@pins/appeals.api",
+		"dev:applications": "turbo run dev --filter=./apps/* --no-cache --parallel --continue",
```

Comment:
Is the spacing off here? prettier should've formatted it, odd.

Reply:
yeah - I can't see the spacing in the local file

Reply:
I think it's tabs verses spaces, and if you have tabs set to 2 space width, it'll look the same.

Our prettierrc has useTabs, but the package json seems to have spaces in. Odd!